react-ui/src/components/Layouts/Dashboard.js

```diff
@@ -62,6 +63,12 @@ class Dashboard extends Component {
           <li>
             <Link to={`/Dashboard/CreateAppointment/${this.state.email}`}>Create Appointment</Link>
           </li>
+
+          <li>
+            <Link to ={`/WorkerAvailableInputPage`}>Worker Input Time Page</Link>
```

Comment:
Hi Kyle,
I think we might need to also keep it with the Dashboard prefix so that these pages have consistent navigation bars and overall layout, i.e., `/Dashboard/InputAvailability`. Also, I would rename the route to have a verb to increase clarity of that specific page.

Reply:
renamed as requested

Reply:
Hi Kyle,
To adjust the routing, add in the relevant **Link** into the Dashboard component (similar to how it is currently done for the `CreateAppointment` route. The difference would be that your route should show when the UserType is `worker` rather than `student`. In addition, you would also need to add a **Route** at the bottom of that file to go to your InputAvailability component.

Reply:
Integrated the pathing with the new dashboard provided.